pkg/apis/eventing/v1alpha1/subscription_types.go

```diff
@@ -171,20 +169,13 @@ type SubscriptionStatus struct {
 	// +patchStrategy=merge
 	Conditions duckv1alpha1.Conditions `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
 
-	// Subscription might be Subscribable. This depends if there's a Result channel
```

Comment:
We should at least keep the Channel here or something. Otherwise you can't chain functions together in a sane manner.  If you change this to Channel ref, then you can subscribe to output of a function. Without this, it's a terrible UX in my experience.

Reply:
I tend to agree with you, I think this is a useful abstraction. But for 0.1, the spec consciously removed it.